hazardous/data/_seer.py

```diff
@@ -0,0 +1,213 @@
+"""Helper to reproduce the SEER dataset transformations as performed in SurvTRACE.
+
+See: https://github.com/RyanWangZf/SurvTRACE/blob/main/data/process_seer.py
+"""
+import pandas as pd
+
+CATEGORICAL_COLUMN_NAMES = [
+    "Sex",
+    "Year of diagnosis",
+    "Race recode (W, B, AI, API)",
+    "Histologic Type ICD-O-3",
+    "Laterality",
+    "Sequence number",
+    "ER Status Recode Breast Cancer (1990+)",
+    "PR Status Recode Breast Cancer (1990+)",
+    "Summary stage 2000 (1998-2017)",
+    "RX Summ--Surg Prim Site (1998+)",
+    "Reason no cancer-directed surgery",
+    "First malignant primary indicator",
+    "Diagnostic Confirmation",
+    "Median household income inflation adj to 2019",
+]
+
+NUMERIC_COLUMN_NAMES = [
+    "Regional nodes examined (1988+)",
+    "CS tumor size (2004-2015)",
+    "Total number of benign/borderline tumors for patient",
+    "Total number of in situ/malignant tumors for patient",
+]
+
+# "xxx" names are placeholders, these columns are not used.
+INDICES_TO_COLUMN_NAMES = {
+    0: "Patient ID",
+    1: "Sex",
+    2: "Year of diagnosis",
+    3: "Site recode ICD-0-3/WHO 2008",
+    4: "COD to site recode",
+    5: "Race recode (W, B, AI, API)",
+    6: "Histologic Type ICD-O-3",
+    7: "ICD-O-3 Hist/behav, malignant",
+    8: "Laterality",
+    9: "Sequence number",
+    10: "Vital status recode (study cutoff used)",
+    11: "ER Status Recode Breast Cancer (1990+)",
+    12: "PR Status Recode Breast Cancer (1990+)",
+    13: "Regional nodes examined (1988+)",
+    14: "xxx",
+    15: "Summary stage 2000 (1998-2017)",
+    16: "xxx",
```

Comment:
Having two different columns with the same `"xxx"` name is likely to cause weird bugs.

Better find descriptive names for those columns, if not possible, then use distinct generic names such as `"Undefined 1"` and `"Undefined 2"`.